### ISSUE TITLE
2010 list confusion

### ISSUE THREAD

**Issue by reupost** [rukayaj/ssp#51]
Sorry Rukaya, I'm getting confused.  

The 2010 list you emailed previously had **689** plant species ('ssp.csv') and **281** animals ('ssp-animals.csv').  The list we've been emailing as 'National Sensitive Species List as at 03 Oct 2017.xlsx' also has the full 970 entries (animals + plants = 970).

But the 2010 list linked here http://biodiversityadvisor.sanbi.org/wp-content/uploads/2015/08/sensitive_species_list_2010.xlsx only has **226** entries.  These are all plants, but its still nowhere near the 689 in the list that we've been working from.

Any ideas?

**Comment by rukayaj** [comment]
TL;DR: ssp.csv is much longer because includes all plants in each genera listed in sensitive_species_list_2010.xlsx. Brenda requested this.

---
Errm.  Hmm. This happened quite a long time ago now, I don't remember what happened.

Ok I was told to work off the "SensitiveTaxa_Plants_merge_12May2010.xlsx" which I think I sent you + Selwyn. It's got 215 entries and it sometimes lists whole genera, so for each genus, I extracted all the species names from BRAHMS and added them manually.  That gives ssp.csv's 689 species. 

I thought the 2010 list http://biodiversityadvisor.sanbi.org/wp-content/uploads/2015/08/sensitive_species_list_2010.xlsx was just be a copy of that 12May2010 spreadsheet with a handful more species added because of splits/name changes/whatever, but it doesn't seem to be (always lists species, never genera). I did a comparison to see what is in ssp.csv and not in sensitive_species_list_2010.xlsx and this is the result (list of genera + count of how many species in each genera):

Aloe | 133
Aloidendron | 1
Anacampseros | 39
Avonia | 6
Bulbine | 1
Cassipourea | 1
Clivia | 7
Dioscorea | 1
Disa | 1
Encephalartos | 37
Euphorbia | 165
Gasteria | 1
Gibbaeum | 1
Haworthia | 157
Haworthiopsis | 3
Hoodia | 9
Isoetes | 1
Lithops | 29
Muraltia | 1
Pachypodium | 4

And now my memory is very vaguely jogged - I think Brenda might have told me to include every species from all genera listed in SensitiveTaxa_Plants_merge_12May2010.xlsx list. I think that must be it, because if I look at the genera list of the sensitive_species_list_2010.xlsx and the genera of ssp.csv they overlap completely apart from 2 genera included in ssp.csv - Aloidendron and Haworthiopsis. Both of which are I think new genera? This article http://pza.sanbi.org/aloidendron-pillansii says Aloidendron is a new genera, and from google Haworthiopsis also seems to be new (2013).

**Comment by rukayaj** [comment]
Keeping track of this issue. 
From Reuben: 

There are 59 names on the list of 226 "SensitiveTaxa_Plants_merge_12May2010.xlsx" sensitive species which have been omitted from the ssp.csv longer list (these are included below).

Query2
--
FULLNAME
Alepidea amatymbica Eckl. &   Zeyh. var. amatymbica
Alepidea amatymbica Eckl. &   Zeyh. var. aquatica (Kuntze) Weim.
Alepidea amatymbica Eckl. &   Zeyh. var. microbracteata Weim.
Aloe brevifolia (Aiton) Haw. [1]
Aloe brevifolia Mill. [2]
Aloe brevifolia Mill. var.   postgenita (Schult. & J.H.Schult.) Baker
Aloe brevifolia Mill. var. serra   (DC.) A.Berger
Aloe krapohliana Marloth var.   dumoulinii Lavranos
Aloe peglerae Schönland
Aloe pillansii L.Guthrie
Ansellia africana Lindl. var.   australis Summerh.
Ansellia africana Lindl. var.   nilotica Baker
Balanites maughamii Sprague   subsp. acuta Sands
Balanites maughamii Sprague   subsp. maughamii
Bowiea volubilis Harv. ex   Hook.f. subsp. volubilis
Cassipourea gummiflua Tul. var.   verticillata (N.E.Br.) J.Lewis
Clivia robusta B.G.Murray, Ran,   de Lange, Hammett, Truter & Swanev.
Conophytum herreanthus   S.A.Hammer subsp. herreanthus
Cotyledon tomentosa Harv. subsp.   ladismithiensis (Poelln.) Toelken
Cotyledon tomentosa Harv. subsp.   tomentosa
Dioscorea sylvatica Eckl. var.   brevipes (Burtt Davy) Burkill
Dioscorea sylvatica Eckl. var.   multiflora (Marloth) Burkill
Dioscorea sylvatica Eckl. var.   paniculata (Dümmer) Burkill
Dioscorea sylvatica Eckl. var.   rehmannii (Baker) Burkill
Dioscorea sylvatica Eckl. var.   sylvatica
Disa lugens Bolus var.   nigrescens (H.P.Linder) H.P.Linder
Euphorbia astrophora Marx
Euphorbia bayeri L.C.Leach
Euphorbia franksiae N.E.Br.
Euphorbia franksiae N.E.Br. var.   franksiae [9]
Euphorbia obesa Hook.f. subsp.   obesa [9]
Euphorbia obesa Hook.f. subsp.   symmetrica (A.C.White, R.A.Dyer & B.Sloane) G.D.Rowley
Euphorbia woodii N.E.Br.
Gasteria nitida (Salm-Dyck) Haw.   var. armstrongii (Schönland) Van Jaarsv.
Haworthia attenuata (Haw.) Haw.
Haworthia attenuata (Haw.) Haw.   var. attenuata
Haworthia attenuata (Haw.) Haw.   var. glabrata Salm-Dyck & M.B.Bayer
Haworthia attenuata (Haw.) Haw.   var. radula (Jacq.) M.B.Bayer
Haworthia bruynsii M.B.Bayer
Haworthia heidelbergensis Gideon   F.Sm. var. heidelbergensis
Haworthia kingiana Poelln.
Haworthia koelmaniorum Oberm.   & D.S.Hardy var. koelmaniorum [9]
Haworthia koelmaniorum Oberm.   & D.S.Hardy var. mcmurtryi (C.L.Scott) M.B.Bayer
Haworthia magnifica Poelln. var.   acuminata (M.B.Bayer) M.B.Bayer
Haworthia magnifica Poelln. var.   dekenahii (G.G.Sm.) M.B.Bayer
Haworthia magnifica Poelln. var.   splendens S.A.Hammer & J.D.Venter
Haworthia marginata (Lam.)   Stearn
Haworthia mirabilis (Haw.) Haw.   subsp. mirabilis
Haworthia mutica Haw. var. nigra   M.B.Bayer
Haworthia pubescens M.B.Bayer   var. pubescens
Haworthia serrata M.B.Bayer
Haworthia subattenuata   (Salm-Dyck) Baker
Haworthia truncata Schönland   var. maughanii (Poelln.) B.Fearn
Haworthia truncata Schönland   var. truncata
Haworthia venosa (Lam.) Haw.
Lithops lesliei (N.E.Br.)   N.E.Br.
Muraltia satureioides DC. var.   salteri (Levyns) & Levyns
Pelargonium reniforme Curtis   subsp. reniforme
Pelargonium reniforme Curtis   subsp. velutinum (Eckl. & Zeyh.) Dreyer

**Comment by rukayaj** [comment]
These names might be caused by a few things:


 - One thing that seems to be common among a lot of them is that they aren’t accepted names. So I probably restricted the extract to accepted names only. 
 - It could also be that some of those names weren’t in BRAHMS at the time I did this work. 
 - During the extraction and mapping I might have made a mistake, weird that it's only 59 affected mind you.

**Comment by rukayaj** [comment]
Suggestion: seeing as we aren't doing any importing any more we just make the original list available on the site and use that alone.

**Comment by reupost** [comment]
Ok.  I'll assemble a version of the short list of 226 species, but with the justifications and threats included for logged-in users to download. That has value I think, but untangling what's happening with the long list is probably not worthwhile.  I'd imagine you're right, the species names have changed in the interim, but for the purposes of staying true to what was defined in 2010 I'm going to stick with those names.

**Comment by rukayaj** [comment]
Great, thanks.

**Comment by reupost** [comment]
Sorted as per issue #30
